client/client.go

```diff
@@ -2356,34 +2316,32 @@ func (c *Client) setGaugeForAllocationStats(nodeID string) {
 	}
 
 	// Emit unallocated
-	unallocatedMem := total.MemoryMB - res.MemoryMB - allocated.MemoryMB
-	unallocatedDisk := total.DiskMB - res.DiskMB - allocated.DiskMB
-	unallocatedCpu := total.CPU - res.CPU - allocated.CPU
-	unallocatedIops := total.IOPS - res.IOPS - allocated.IOPS
+	unallocatedMem := total.Memory.MemoryMB - res.Memory.MemoryMB - allocated.Flattened.Memory.MemoryMB
+	unallocatedDisk := total.Disk.DiskMB - res.Disk.DiskMB - allocated.Shared.DiskMB
+	unallocatedCpu := total.Cpu.CpuShares - res.Cpu.CpuShares - allocated.Flattened.Cpu.CpuShares
 
 	if !c.config.DisableTaggedMetrics {
 		metrics.SetGaugeWithLabels([]string{"client", "unallocated", "memory"}, float32(unallocatedMem), c.baseLabels)
 		metrics.SetGaugeWithLabels([]string{"client", "unallocated", "disk"}, float32(unallocatedDisk), c.baseLabels)
 		metrics.SetGaugeWithLabels([]string{"client", "unallocated", "cpu"}, float32(unallocatedCpu), c.baseLabels)
-		metrics.SetGaugeWithLabels([]string{"client", "unallocated", "iops"}, float32(unallocatedIops), c.baseLabels)
```

Comment:
so iops is dead as a resource for now?

Reply:
Yeah.. It wasn't being used, so not modeling it for now